front/packages/ui/src/details/episode.tsx

```diff
@@ -99,7 +103,10 @@ export const EpisodeBox = ({
 							borderStyle: "solid",
 						},
 						more: {
-							display: "none",
+							opacity: 0,
```

Comment:
I don't think this is used, is it?

Reply:
Yes it is: the `more` class now sets the opacity of an element to 1 (instead of setting `display` to none) when the parent is hovered.
However, the `less` class is not used. Removing it right away.